### ISSUE TITLE
md-tab not showing any text

### ISSUE THREAD

**Issue by theunreal** [angular/material2#1651]
#### What is the expected behavior?
Texts appear in a tab

#### What is the current behavior?
Texts do not appear in a tab

#### What are the steps to reproduce?
I really have no idea why it happens.
I just tried to use tabs regulary in my app.

I really have no idea why the display is not working, and not showing any texts, my app is too big for a plunker. I also have no idea how to reproduce it. 

I have simply this:
```
<md-tab-group>
  <md-tab label="One">
    <h1>One</h1>
        Test
    </md-tab>
</md-tab-group>

```

Outputs:
![image](https://cloud.githubusercontent.com/assets/6171883/19830947/b3c8a7b2-9e07-11e6-9ac2-2e36362b9738.png)

#### What is the use-case or motivation for changing an existing behavior?
Md tab bug fix

#### Which versions of Angular, Material, OS, browsers are affected?
Angular: 2.0.2,
"@angular/material": "^2.0.0-alpha.9-3",
OS: Windows
Browers: Chrome

**Comment by RoxKilly** [comment]
That may not be the correct usage for `2.0.0-alpha.9-3`

Do instead:
```
<md-tab-group>
  <md-tab>
     <template md-tab-label>One</template>
    <template md-tab-content>
      Test
    </template>
  </md-tab>
</md-tab-group>
```
Your code uses the simple API but I don't think that it's part of the `alpha 9-3` release

**Comment by theunreal** [comment]
Thanks @RoxKilly ! The documentation mislead me.

**Comment by vinagreti** [comment]
"Your code uses the simple API but I don't think that it's part of the alpha 9-3 release"
I don't think so. I think that the two approaches are valid but for some reason the simple API is not working in `alpha.9-3`.
I think it is really a bug.
Could someone from the core team ensure that the simple API was removed and we should use only the `<template md-tab-label>One</template>` ?

Thanks!

**Comment by RoxKilly** [comment]
@vinagreti the Plunker you linked to does not use the released `alpha 9-3`. It uses the development version (look at `systemjs.config.js`)

I wasn't implying that the simple API has been *removed* from 9-3, but rather that it has not yet made its way to release (ie. it will be in a later version because right now it's just in the development channel)

**Comment by vinagreti** [comment]
Sure, because of that (Dev version) I thought that simple API was not removed from material but it was indeed removed from alpha-9.3.

Thanks for your quick reply!

**Comment by RoxKilly** [comment]
@theunreal @vinagreti the code in the OP works on version  `alpha 10`

**Comment by jacastellanosc** [comment]
Hi, I solved changing the CSS the angular-material v1.0.0 by v1.1.0 

This is the new CSS: https://ajax.googleapis.com/ajax/libs/angular_material/1.1.0/angular-material.min.css

**Comment by ZiruZanrgeiff** [comment]
@jacastellanosc man this bug was reported in 2016.

**Comment by jacastellanosc** [comment]
Sorry, until now I occurred me. And today it showed itself  the error in my
code.




*​____________________​​​Jimmy Castellanos C*
Celular: +57 304 614 4224
Skype: jacastellanoscc
www.lyons-sas.com

2018-02-26 9:36 GMT-05:00 Ronildo dos Santos <notifications@github.com>:
